StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentElementSnapshotTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EmbeddedPaymentElementSnapshotTests.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 10/16/24.
+//
+
+import StripeCoreTestUtils
+@_spi(STP) @testable import StripePayments
+@_spi(EmbeddedPaymentElementPrivateBeta) @testable import StripePaymentSheet
+@testable import StripePaymentsTestUtils
+@_spi(STP) @testable import StripeUICore
+import XCTest
+
+class EmbeddedPaymentElementSnapshotTests: STPSnapshotTestCase, EmbeddedPaymentElementDelegate {
```

Comment:
Are we planning to add more tests to this file? If so do we have a ticket or will that happen as a follow up?

Reply:
Yep will add more tests with the next PRs (see bullets in summary)!

Reply:
ACK, saw bullets just didn't think they'd be snapshot tests? Haven't thought much about it though.

Reply:
Oh, missed "in this file".  I think we'll eventually add but I don't have any specific tests in mind for `update`.